src/pages/policy/PolicyRightSidebar.jsx

```diff
@@ -93,6 +93,133 @@ function TimePeriodSelector(props) {
   );
 }
 
+const behaviouralResponseReforms = {
+  uk: {
+    "gov.simulation.capital_gains_responses.elasticity": {
+      "2024-01-01.2100-01-01": -0.7,
+    },
+    "gov.simulation.labor_supply_responses.income_elasticity": {
+      "2024-01-01.2100-01-01": -0.05,
+    },
+    "gov.simulation.labor_supply_responses.substitution_elasticity": {
+      "2024-01-01.2100-01-01": 0.25,
+    },
+  },
+  us: {},
```

Comment:
I think it might make sense here to just add the US's "Apply CBO elasticities" reform option. Is that at all usable for the UK?